pkg/deploy/elbv2/load_balancer_manager.go

```diff
@@ -186,20 +193,52 @@ func (m *defaultLoadBalancerManager) updateSDKLoadBalancerWithSecurityGroups(ctx
 	}
 	desiredSecurityGroups := sets.NewString(awssdk.StringValueSlice(securityGroups)...)
 	currentSecurityGroups := sets.NewString(awssdk.StringValueSlice(sdkLB.LoadBalancer.SecurityGroups)...)
-	if desiredSecurityGroups.Equal(currentSecurityGroups) {
+
+	if desiredSecurityGroups.Equal(currentSecurityGroups) && (resLB.Spec.Type != elbv2model.LoadBalancerTypeNetwork || resLB.Spec.SecurityGroupsInboundRulesOnPrivateLink == nil) {
```

Comment:
is there an "Describe" API that shows the "EnforceSecurityGroupInboundRulesOnPrivateLinkTraffic" on a existing LB? 
If not, we may need to make this call on every Ingress reconcile, which feels unnecessary. 
Shall we consider use a cache to overcome it? i.e. cache the state of "EnforceSecurityGroupInboundRulesOnPrivateLinkTraffic" per LB if ELB didn't offer a describe API.